### PR TITLE
Add support for x-amz-bypass-governance-retention in RemoveObjects

### PR DESCRIPTION
**Pull request by nitisht**

### CODE REVIEW

File: api-remove.go [hunk 1]
```diff
@@ -179,71 +179,107 @@ func (c Client) RemoveObjectsWithContext(ctx context.Context, bucketName string,
...
-			if resp != nil {
-				if resp.StatusCode != http.StatusOK {
-					e := httpRespToErrorResponse(resp, bucketName, "")
-					errorCh <- RemoveObjectError{ObjectName: "", Err: e}
-				}
+		// Build headers.
+		headers := make(http.Header)
+		if opts.GovernanceBypass {
+			// Set the bypass goverenance retention header
+			headers.Set("x-amz-bypass-governance-retention", "True")
```
Comment (harshavardhana) [inline api-remove.go:250]: "True" is "true"

Comment (harshavardhana) [reply api-remove.go:250]: also lets `x-amz-bypass-governance-retention` make this a constant

File: api-remove.go [hunk 2]
```diff
@@ -179,71 +179,107 @@ func (c Client) RemoveObjectsWithContext(ctx context.Context, bucketName string,
...
+		}
+
+		// Generate remove multi objects XML request
+		removeBytes := generateRemoveMultiObjectsRequest(batch)
+		// Execute GET on bucket to list objects.
+		resp, err := c.executeMethod(ctx, "POST", requestMetadata{
```
Comment (harshavardhana) [inline api-remove.go:256]: use http.MethodPost instead of "POST"

**Comment by nitisht** [comment]
Done, PTAL

**Pull request merged**